Editing some Greek inscriptions from Catania

martafogagnolo1 opened the pull request:
ISic000361, ISic000647, ISic000651, ISic000714, ISic001276, ISic001280, ISic001282, ISic001283, ISic001288, ISic003006

JonPrag commented on inscriptions/ISic000647.xml:
```diff
@@ -200,28 +205,56 @@
+                        break="no"/><supplied reason="lost">ζω</supplied></w> <supplied reason="lost">τὸν</supplied> <w><supplied reason="lost">Παντο</supplied>κράτορα</w> καὶ
+                    <lb n="7"/><gap reason="lost" extent="unknown" unit="character"/><supplied reason="lost">τὸν</supplied> <w><supplied reason="lost">μέλλ</supplied>οντα</w> <w>α<hi rend="ligature">ἰῶ</hi>να</w>
+                    <lb n="8"/><gap reason="lost" extent="unknown" unit="character"/><w><supplied reason="lost">μη</supplied><unclear>δ</unclear>ὲν'</w> <w>ἀνῦξαι</w>
+                    <lb n="9"/><gap reason="lost" extent="unknown" unit="character"/><w><supplied reason="lost" cert="low">ἐκεί</supplied>νων</w> <w>κατ<supplied reason="lost">ά</supplied><lb n="10" 
```
add white space before first `<w>`

JonPrag commented on inscriptions/ISic000647.xml:
```diff
@@ -200,28 +205,56 @@
+                    <lb n="7"/><gap reason="lost" extent="unknown" unit="character"/><supplied reason="lost">τὸν</supplied> <w><supplied reason="lost">μέλλ</supplied>οντα</w> <w>α<hi rend="ligature">ἰῶ</hi>να</w>
+                    <lb n="8"/><gap reason="lost" extent="unknown" unit="character"/><w><supplied reason="lost">μη</supplied><unclear>δ</unclear>ὲν'</w> <w>ἀνῦξαι</w>
+                    <lb n="9"/><gap reason="lost" extent="unknown" unit="character"/><w><supplied reason="lost" cert="low">ἐκεί</supplied>νων</w> <w>κατ<supplied reason="lost">ά</supplied><lb n="10" 
+                        break="no"/><supplied reason="lost" cert="low">θεσιν</supplied></w><gap reason="lost" extent="unknown" unit="character"/>
```
add white space after final closing `<w>`

JonPrag commented on inscriptions/ISic000647.xml:
```diff
@@ -200,28 +205,56 @@
+                </listApp>
+            </div>
+            <div type="translation" xml:lang="en" resp="#martaf">
+                <p><gap reason="lost"/>all the world <gap reason="lost"/><supplied reason="subaudible">of so and so</supplied> and of Basilis. They lived perfectly and blamelessly with me fourteen years <gap reason="lost"/>lived twenty-eight years. I beseech in the name of the Almighty <gap reason="lost"/>for all eternity <gap reason="lost"/>let no one open <supplied reason="lost">the tomb</suppl ...
```
simply: [---] all the world [--- of (name)] and of Basilis.
and so on - no mark-up, just written out in plain text.

JonPrag commented on inscriptions/ISic000647.xml:
```diff
@@ -232,28 +265,45 @@
+                    </bibl>
+                </listBibl>
+                <listBibl type="discussion">
+                    <bibl type="corpus" n="IMusCatania">
```
not clear what the second ref to IMusCatania is here in discussion? page number? corpus number? something distinct from the entry for no. 212?

JonPrag commented on inscriptions/ISic000647.xml:
```diff
@@ -232,28 +265,45 @@
+                        <author>Feissel</author>
+                        <date>1980</date>
+                        <citedRange>464-470</citedRange>
+                        <ptr target=""/>
```
We can walk through the process for creating bibliographic references in Zotero and generating the URI, if necessary.

JonPrag reviewed (commented):
Hi Marta,
I've looked at the first two. Would it be useful (a) for you to look and see if the comment process in GitHub makes sense to you and is easily accessible, and (b) if we then have a quick chat about things that come up there and anything else, before I go through the rest?

JonPrag reviewed (commented):
I'm halfway through editing these at last - will follow up as soon as I'm done!

JonPrag reviewed (approved):
Dear Marta
very many apologies indeed for having sat on these for so long.  Rather than engage in a long process of back and forth, I have made minor edits on all (but not anything substantive I think!), and incorporated the files - huge thanks for doing these.
On the specific issue of msParts, the essential criteria as regards whether we implement them or not come down to whether it is necessary to document the multiple parts in a machine readable way - for an inscription of multiple joining fragments, this is rarely necessary, and even if we have the dimensions of individual fragments, it is simpler (and much less cumbersone)  just to document the various dimensions in free text in the support description. We only make the effort for msParts if there is different provenance history (and it is useful to record it in a searchable way) or if, e.g., the connection between the fragments is uncertain or very liable to change. Otherwise, it really seems like unnecessary effort on all sides!

martafogagnolo1 commented:
Dear Jonathan,
thank you for having made the edits and incorporated the files! And thanks
also for the explanation regarding msPart! We’ll be in touch soon with
other inscriptions!
All best,
Marta

Il giorno lun 9 mag 2022 alle ore 19:03 Jonathan Prag <
***@***.***> ha scritto:

> ***@***.**** approved this pull request.
>
> Dear Marta
> very many apologies indeed for having sat on these for so long. Rather
> than engage in a long process of back and forth, I have made minor edits on
> all (but not anything substantive I think!), and incorporated the files -
> huge thanks for doing these.
> On the specific issue of msParts, the essential criteria as regards
> whether we implement them or not come down to whether it is necessary to
> document the multiple parts in a machine readable way - for an inscription
> of multiple joining fragments, this is rarely necessary, and even if we
> have the dimensions of individual fragments, it is simpler (and much less
> cumbersone) just to document the various dimensions in free text in the
> support description. We only make the effort for msParts if there is
> different provenance history (and it is useful to record it in a searchable
> way) or if, e.g., the connection between the fragments is uncertain or very
> liable to change. Otherwise, it really seems like unnecessary effort on all
> sides!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ISicily/ISicily/pull/216#pullrequestreview-966529298>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUYXHOHKAOH7MIG6WCGDIQ3VJFAPLANCNFSM5IT44ZWA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
